bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -60,10 +60,20 @@ You will get the most out of this lesson if you follow along with the examples a
 If you are using a computer with running iOS (i.e. a Mac) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.
 
 <div class = "important">
+<b style="color: rgb(var(--color-highlight));">Important note</b><br>
+
 We want to be able to search, move, and rename files during this module, but don't want to do that with your important files. Therefore we will set up a little directory with a few files to experiment with. You can safely delete the whole thing afterwards if you want.
```

Comment:
Good point, it doesn't exactly fit any of the highlight boxes... I'll update it in all of them.

Reply:
However I want to keep it as an important box for the Bash 102 module, since that is the first time people will be moving stuff around using the command line and they might be a bit scared about screwing around with their own files... What do you think?

Reply:
Let's reformulate this text and make it a "care" box instead.